<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gcd_widget</class>
 <widget class="QWidget" name="gcd_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: white;
background-color: black;</string>
  </property>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Số 1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nhapSo_1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white; 
color: black;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Số 2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nhapSo_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white; 
color: black;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_giai">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #EEA10C;
color: white;
border-radius: 50px;</string>
   </property>
   <property name="text">
    <string>Tìm Ước chung lớn nhất</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_ketqua">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black; 
background-color: white;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
